docs/sidebars.ts

```diff
@@ -3,83 +3,94 @@ import type {SidebarsConfig} from "@docusaurus/plugin-content-docs";
 const sidebars: SidebarsConfig = {
   tutorialSidebar: [
     "index",
-    "introduction",
-    {
-      type: "doc",
-      label: "Security",
-      id: "security",
-    },
-    {
-      type: "category",
-      label: "Getting Started",
-      items: ["getting-started/quick-start", "getting-started/installation", "getting-started/starting-a-node"],
-    },
-    "data-retention",
-    {
-      type: "category",
-      label: "Beacon Node",
-      items: [
-        "beacon-management/beacon-cli",
-        "beacon-management/networking",
-        "beacon-management/mev-and-builder-integration",
-        "beacon-management/syncing",
-      ],
-    },
     {
       type: "category",
-      label: "Validator",
+      label: "Run A Node",
       items: [
-        "validator-management/validator-cli",
-        "validator-management/vc-configuration",
-        "validator-management/external-signer",
+        "run/getting-started/quick-start",
+        "run/getting-started/installation",
+        {
+          type: "category",
+          label: "Beacon node",
+          collapsed: false,
+          items: [
+            "run/beacon-management/starting-a-node",
+            "run/beacon-management/beacon-cli",
+            "run/beacon-management/data-retention",
+            "run/beacon-management/networking",
+            "run/beacon-management/mev-and-builder-integration",
+            "run/beacon-management/syncing",
+          ],
+        },
+        {
+          type: "category",
+          label: "Validator Client",
+          collapsed: false,
+          items: [
+            "run/validator-management/vc-configuration",
+            "run/validator-management/validator-cli",
+            "run/validator-management/external-signer",
+          ],
+        },
+        {
+          type: "category",
+          label: "Logging and Metrics",
+          collapsed: false,
+          items: ["run/logging-and-metrics/prometheus-grafana", "run/logging-and-metrics/client-monitoring"],
+        },
+        {
+          type: "category",
+          label: "Discv5 Bootnode",
+          collapsed: false,
+          items: ["run/bootnode/bootnode-cli"],
+        },
       ],
     },
     {
       type: "category",
-      label: "Bootnode",
-      items: ["bootnode/bootnode-cli"],
-    },
-    {
-      type: "category",
-      label: "Light Client and Prover",
+      label: "Developer Tools",
+      collapsed: false,
       items: [
-        "lightclient-prover/lightclient-cli",
         {
-          type: "doc",
-          label: "Light Client",
-          id: "lightclient-prover/lightclient",
+          type: "category",
+          label: "Lodestar Light Client",
```

Comment:
we don't need to prefix this with Lodestar, it's redundant, and we don't do it for anything else